system/group.py

```diff
@@ -36,6 +36,12 @@
         required: false
         description:
             - Optional I(GID) to set for the group.
+    force:
+        required: false
+        default: false
```

Comment:
@b3rn, I think `default: "no"` would be more consistent with other default values.